apps/documentation/docs/getting-started/serverless.md

```diff
@@ -2,24 +2,286 @@
 sidebar_position: 0
 ---
 
-# Serverless
+# Create a Serverless Service
 
-To get started with running a version of the leadership site locally follow these steps:
+To setup a new serverless service in this repo, follow these steps:
 
-### Setup
+### Install Serverless Framework CLI
 
-- Clone the repo `git clone https://github.com/acmutd/leadership.git`
-- Fill in the values in `.env.local.example` and rename it to `.env.local`. Reach out to an existing contributor to get these values.
+- To install Serverless Framework CLI, run the following:
 
-Note: You can also find the environment variables via [Doppler](https://doppler.com). Open the Doppler website, sign in with your ACM account and open the leadership project. Then copy the variables in the `dev` config to your local `.env.local` file. Currently, the leadership site uses Doppler to fetch environment variables for its production vercel deployment. It is not configured to pull the environment variables from Doppler for local workloads yet.
+```
+npm i -g serverless
+```
 
-### Launch
+### Directory Setup
 
-- Run `npm install`
-- Run `npm run dev`
+- Create a new folder with the name of the service inside the `services` directory
+  - e.g: If your service name is `email`, then create the new directory `services/email`.
+- Run the following command and follow the prompt:
+
+```
+npm init -w ./services/email
```

Comment:
```suggestion
npm init --scope @acmutd -w ./services/email
```

Reply:
also run the command `Monorepo: sync workspace features` in the VSCode command palette to update the file sidebar config with new package